Documentation/config.xml

```diff
@@ -42,6 +43,8 @@
     <topic id="Localization" name="Localization" filename="localization.md" />
     </topic>
 
+    <topic id="Distributing games" name="Distributing games" filename="distributing_games.md"/>
```

Comment:
Should we call this 'Package games for distribution` or something instead? We don't go into actual storefronts and stuff.

Reply:
You're right.